package.json

```diff
@@ -10,9 +10,14 @@
     "node": "^16.6.1",
     "npm": ">=7.20.0"
   },
-  "main": "index.js",
+  "exports": {
+    "./style/standard": "./style/standard.js",
+    "./style/verbose": "./style/verbose.js",
+    "./correctness/all": "./correctness/all.js"
```

Comment:
We could potentially just do `/correctness` until the day that we add multiple correctness scripts. This has the added benefit that if we ever add some optional correctness rules, we don't have a breaking change where suddenly users must change from `correctness/all` to `correctness/base` or something like that. 

Follow the future proof vein, I propose:
 - Rename `correctness/all.js` to `correctness/index.js`
 - Export just `./correctness` so users dont have to `./correctness/index.js`
 - If we add more correctness rules in the future they will likely be extensions to `index.js`, accessible via `correctness/new-rule.js`